modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -56,20 +61,27 @@ spec:
     resources:
       requests:
         storage: <fs_volume_size> <2>
-  osImages: <3>
-    - openshiftVersion: "<ocp_version>" <4>
-      version: "<ocp_release_version>" <5>
-      url: "<iso_url>" <6>
-      rootFSUrl: "<root_fs_url>" <7>
+  imageStorage:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: <is_volume_size> <3>      
```

Comment:
Could consider more explicit sub values, e.g. `<image_storage_volume_size>`. On first scan, I thought maybe this field expects a Boolean value.